RELEASING.md

```diff
@@ -3,6 +3,7 @@
 1. Communicate that a release is about to be released in [Gitter Akka Dev Channel](https://gitter.im/akka/dev), so that no new Pull Requests are merged
 1. Add a release notes entry in docs/src/main/paradox/release-notes/. As a helper run
 `scripts/commits-for-release-notes.sh <last-version-tag>` which will output a list of commits grouped by submodule, and the closed issues for this milestone.
+1. Add the released version to `project/MiMa.scala` to the `mimaPreviousArtifacts` key.
```

Comment:
The problem with putting that step here is, that you cannot commit it yet, because mima cannot find the newly created release yet. On the other hand, I guess that putting it in line 4 might increase the probability that the releaser actually reads the line :)

Reply:
Ah of course...